client/src/index.css

```
@@ -201,4 +201,75 @@ body {
+  .text-lime-600.dark\:text-lime-400 {
+    color: #15803d !important;
+  }
+
+  /* Score circle and chart improvements */
+  [class*="dark:stroke-"] {
+    stroke: #030712 !important;
+  }
+
+  /* Page breaks */
+  .bg-white.dark\:bg-stone-900 {
+    page-break-inside: avoid;
```

Stylelint (17.11.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)